common/src/configuration/utils.rs

```diff
@@ -80,6 +80,7 @@ pub fn default_config(bootstrap: &ConfigBootstrap) -> Config {
     cfg.set_default("wallet.transaction_direct_send_timeout", 600).unwrap();
     cfg.set_default("wallet.transaction_broadcast_send_timeout", 600)
         .unwrap();
+    cfg.set_default("wallet.dust_collector_enabled", false).unwrap();
```

Comment:
The default and sample configs should align. But `false` is the correct answer here. :)